.gitlab-ci.yml

```diff
@@ -11,6 +11,7 @@ stages:
   - METRICS-CHECK
   - TEAMING-CHECK
   - TOPOLOGY-CHECK
+  - OPENEBS-DOP-INSTALL-CHECK
```

Comment:
Why is a new stage required?

Reply:
@uditgaurav we can put this in the same stage - `OPENEBS-INSTALL-CHECK` and it can run parallel to other jobs in it.

Reply:
okay